content/en/Getting started/sign-in.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sign In to Cobalt"
+linkTitle: "Sign In"
+weight: 9
+description: >
+  Start the pentest process. Sign in to the Cobalt app.
+---
+
+{{% pageinfo %}}
+This page assumes that you've received a welcome email from Cobalt.
+{{% /pageinfo %}}
+
+Even if you haven't yet purchased Cobalt credits, this page (and document)
+can help you visualize how you can set up a pentest with the Cobalt UI.
+
+You've just received an email with the following title:
+
+```
+Welcome to the Cobalt Platform: Let's Get Started
+```
+
+Open the email. It should include a link to Get Started:
+
+![Cobalt Welcome Email](/WelcomeEmail.png "Get Started")
+
+Now you can:
+
+1. Select the link in your email.
+
+1. From the webpage that appears, create a password. Follow the complexity
+   requirements on the screen. We require passwords with at least:
+   - Eight (8) characters
+   - One (1) uppercase letter
+   - One (1) lowercase letter
+   - One (1) digit
+
```

Comment:
one other requirement, which i'm not sure we need to mention, is that you cannot use any of the last 10 previous passwords.  specifying that in the docs might provide more information than what is really needed, and give clues to anyone attempting to hack our authentication?

Reply:
I'm tempted to include a note... but I'd like to find the code file to watch (my search foo failed me here). @reginandrade can you point me to the code that specifies "last 10 previous passwords"